client/src/components/Event.test.tsx

```diff
@@ -0,0 +1,20 @@
+import { render, screen } from "@testing-library/react";
+import Event from "./Event";
+import { EventProps } from "../types";
+
+const data: EventProps = {
+  location: "Karlin",
+  id: "rust-meetup",
+  title: "Rust Meetup",
+  dates: []
+};
+
+test("event name and location displayed", () => {
+  render(<Event {...data} />);
+  const title = screen.getByText("Event: Rust Meetup");
+  expect(title).toBeInTheDocument();
```

Comment:
tohle je malicko redundantni, protoze `getByText` vyhodi chybu (a tim zfailuje test), pokud ten element neni v documentu. Takze to, ze radek 14 projde uz zarucuje, ze ten element tam existuje. Ale je to relativne casty "ukaz"/styl testu

Mozna vice "ciste" by bylo pouzit jinou metodu:
```
  const title = screen.queryByText("Event: Rust Meetup");

```